sapphire/tests/transformations/test_celestial.py

```diff
@@ -159,5 +161,323 @@ def test_against_jpl_horizons(self):
+
+    zenrange = [0, np.pi]
+    plt.subplot(211)
+    plt.title('Zenith')
+    plt.axis(zenrange*2)
+    plt.xlabel('New (Astropy)')
+    plt.ylabel('Old')
+
+    # Make figure and add 1:1 trendline
+
+    plt.plot([co[0] for co in etoha], [co[0] for co in etoh], 'r.', zenrange, straight(zenrange), '-')
+
+    plt.subplot(212)
+    plt.title('Azimuth')
+    azrange = [-np.pi, np.pi]
+    plt.axis(azrange*2)
+    plt.xlabel('New (Astropy)')
+    plt.ylabel('Old')
+    # Make figure and add 1:1 trendline
+    plt.plot([co[1] for co in etoha], [co[1] for co in etoh], 'b.', azrange, straight(azrange), '-')
+    plt.tight_layout() # Prevent titles merging
+    plt.subplots_adjust(top=0.85)
+
+    # Make histogram of differences
+    plt.figure(2)
+    nieuw = (np.array(etoh)-np.array(etoha))/2/np.pi*360*3600 # Take difference and convert to arcsec
+    plt.hist([i[0] for i in nieuw], bins = 20)
+    plt.title('Zenith Old-New Error (equatorial_to_zenithazimuth/horizontal)')
+    plt.xlabel('Error (arcsec)')
+    plt.ylabel('Counts')
+
+    plt.figure(3)
+    plt.hist([i[1] for i in nieuw], bins=20)
+    plt.title('Azimuth Old-New Error (equatorial_to_zenithazimuth/horizontal)')
+    plt.xlabel('Error (arcsec)')
+    plt.ylabel('Counts')
+
+    # Make figs hor - > eq
+
+    plt.figure(4)
+    plt.suptitle('RA/DEC  correlation in rads (horizontal_to_equatorial)')
+    altrange = [-0.5*np.pi, 0.5*np.pi]
+    plt.subplot(211)
+    plt.title('Declination')
+    plt.axis(altrange * 2)
+    plt.xlabel('New (Astropy)')
+    plt.ylabel('Old')
+    # Make figure and add 1:1 trendline
+    plt.plot([co[1] for co in htoea], [co[1] for co in htoe], 'r.', altrange, straight(altrange), '-')
+
+    plt.subplot(212)
+    plt.title('Right Ascension')
+    azrange = [0, 2*np.pi]
+    plt.axis(azrange * 2)
+    plt.xlabel('New (Astropy)')
+    plt.ylabel('Old')
+    # Make figure and add 1:1 trendline
+    plt.plot([co[0] for co in htoea], [co[0] for co in htoe], 'b.', azrange, straight(azrange), '-')
+    plt.tight_layout()  # Prevent titles merging
+    plt.subplots_adjust(top = 0.85)
+
+    # Make histogram of differences
+    plt.figure(5)
+    nieuw = (np.array(htoe) - np.array(htoea)) / 2 / np.pi * 360 * 3600  # Take difference and convert to arcsec
+    plt.hist([i[1] for i in nieuw], bins=20)
+    plt.title('Declination Old-New Error (horizontal_to_equatorial)')
+    plt.xlabel('Error (arcsec)')
+    plt.ylabel('Counts')
+
+    plt.figure(6)
+    nieuw = (np.array(htoe) - np.array(htoea)) / 2 / np.pi * 360 * 3600  # Take difference and convert to arcsec
+    plt.hist([i[0] for i in nieuw], bins=20)
+    plt.title('Right Ascension Old-New Error (horizontal_to_equatorial)')
+    plt.xlabel('Error (arcsec)')
+    plt.ylabel('Counts')
+
+    plt.show()
+    return
+
+try:
+    # This try-except block contains a pyephem accuracy benchmarking function.
+    # It uses this structure to accommodate people without pyephem.
+    import ephem
+    def pyephem_comp():
```

Comment:
This 'test'/comparison/benchmark function should be in the scripts directory ( `/sapphire/scripts/...` )